blog/rome-incentives/README.md

```diff
@@ -0,0 +1,225 @@
+# Post
+
+#### Title
+
+The Rome Incentive Structure
+
+#### Purpose
+
+Inform our community about Rome's incentive structure.
+
+#### url
+
+blog.joystream.org/rome-incentives/
+
+#### Cover
+
+<p align="center"><img src="rome-incentives.png"></p>
+
+#### Lead
+
+Rome offers a new incentive structure for our testnet participants.
+
+#### Body
+
+As with Sparta, Athens and Acropolis we plan to pay users in monero (XMR) to participate on our upcoming testnet release, codenamed Rome. Our fifth testnet ambitiously introduces two new paid roles which we are extremely excited about!
+
+Executive Summary
+-----------------
+
+- `Content Creators` receive up to $0.60 per qualifying item of content uploaded\
+- `Content Curators` earn either $30 or $60 per week, depending on workload\
+- `Storage Providers` will share $300 per week, plus $0.025/GB per week\
+- `Validators` are competing for their share of $50 per week\
+- `Council Members` are paid $10 for getting elected, with a $10 bonus\
+- `Bug Reporters` can continue to earn monero for reporting and/or fixing bugs
+
+Who, Why, What and How?
+-----------------------
+
+As with previous incentivized testnets, anyone who wants to can participate on Rome and earn monero, but there is some extra help and guidance for those that ask for it in our [Telegram group](https://t.me/JoyStreamOfficial).
+
+The philosophy behind why was discussed in detail in a [previous post](https://blog.joystream.org/pay-for-play/). In short, we are paying to prepare active participants for an autonomous platform and test the incentive structure with "real money".
+
+What you can do to qualify for getting paid on Rome is described below. More information, guides on how to get setup and extra support can be found in our helpdesk repo on GitHub.
+
+Paid Roles
+----------
+
+- Content Creator
+- Content Curator
+- Storage Provider
+- Validator
+- Council Member
+- Bug Reporter
+
+If you find one or more of these roles interesting, or want to learn about future roles on the network, we encourage you to visit the new [roles](https://joystream.org/roles) section on our website. Here you can browse through them and get a sense of what they will mean to the network, along with responsibilities and requirements.
+
+You can also sign up for role specific newsletters that will keep you updated on development and incentives!
+
+Content Creators
+----------------
+
+![](Content%20Creators.png)
+
+<i>Compete for $350 per week as a content creator</i>
```

Comment:
I think we have to revise the thinking here. Something like paying for first `n` videos a week, but always payout for original content?